data/rulei/rulei.txt

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) 2022 by Saugia
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE. See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along with
+# this program. If not, see <https://www.gnu.org/licenses/>.
+
+mission "Relic of the Past"
+	invisible
+	landing
+	to complete
+		never
+	npc save
+		government "Rulei"
+		personality heroic vindictive staying mute unconstrained frugal
+		system "Kanguwa"
+		ship "vyu-Ira" "Solemnity"
+
+mission "Ira Contact"
+	invisible
+	landing
+	to complete
+		never
+	on enter Kanguwa
+		fail
+		log "Factions" "Rulei" `The Rulei were likely an ancient, technologically advanced civilization from long ago.`
+			`They communicate through some sort of invasive psionic ability.`
+		conversation
+			`The second you jump into this system you begin to experience a strange, uncomfortable sensation. You feel an overwhelming sense of emptiness, but at the same time the intensity of this feeling suggests that something must be there. You barely have enough time to try and make sense of this oddity as only seconds later you start to hear ominous whispering come from behind you.`
+			choice
+				`	(Turn around.)`
+			`	You turn your head around to look, but the source of the whispers remains at the back of your head, like an itch that can't be reached. You can't even remotely understand the language, and the whispers progressively increase in volume. The louder they get, the more there are. They begin to come from various directions around you, as though they're tugging at your mind from several angles at once. It sounds like hundreds of differently pitched voices are speaking all at once now, and your senses all feel overloaded. Just before you think you can't take anymore, everything falls completely silent.`
```

Comment:
I think this is too many "whispers" in a row, which is what I was trying to avoid previously (three sentences in a row). I'll try to think of a way to change it, but the rest is good.